realm/realm-library/src/objectServer/java/io/realm/internal/objectstore/OsAppCredentials.java

```diff
@@ -51,10 +50,6 @@ public static OsAppCredentials apiKey(String key) {
         return new OsAppCredentials(nativeCreate(TYPE_API_KEY, key));
     }
 
-    public static OsAppCredentials serverApiKey(String key) {
```

Comment:
This wasn't really used and not exposed in the SDK. Removed from core in https://github.com/realm/realm-core/issues/5914